src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -1745,6 +1745,7 @@ public enum DateTimeKind
         public int Second { get { throw null; } }
         public long Ticks { get { throw null; } }
         public System.TimeSpan TimeOfDay { get { throw null; } }
+        public int TotalOffsetMinutes { get { throw null; } }
```

Comment:
What is the primary purpose of this new property? From the issue description, it seems like it's purely about performance? If so, do we have numbers to validate it is in fact meaningfully better than Offset in the scenarios it'll be used? And out of all the places we use Offset, should any be using this instead?

Reply:
I talk about it a bit [here](https://github.com/dotnet/runtime/issues/52081#issuecomment-829654064) in the proposal.

It's been quite a long time- I don't have the exact numbers anymore unfortunately. There are a number of places in the code where we could use this.

https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/DateTimeOffset.cs,1bc976bb59f3dd6e,references

Reply:
One interesting data point is that, at least in Azure scenarios, `DateTimeOffset`s are _usually_ UTC. In the index link above there are a few places where we're comparing the `Offset` to `TimeSpan.Zero`.

Reply:
I would have liked to see this PR use the new API. Who's following up to do that?

Reply:
What do you mean by this PR use the new API?

Reply:
Code in dotnet/runtime production src calling it.

Reply:
I searched the who runtime repo and the only files can use the new property are:

runtime\src\libraries\System.Formats.Cbor\src\System\Formats\Cbor\Writer\CborWriter.Tag.cs
runtime\src\libraries\System.Private.Xml\src\System\Xml\Core\XmlWriter.cs
runtime\src\libraries\System.ServiceModel.Syndication\src\System\ServiceModel\Syndication\Atom10FeedFormatter.cs
runtime\src\libraries\System.ServiceModel.Syndication\src\System\ServiceModel\Syndication\Rss20FeedFormatter.cs

If it is worth it, I can try to submit a PR for these.

Reply:
Thanks.  I think if it's worth adding the property, it's worth using it, assuming it makes the usage simpler/faster/etc.  I'm pushing on this as I do any time we add API because it helps to validate whether the API is designed correctly and actually worthwhile.

Reply:
Sorry if I merged early. I'll try to submit a PR for that in the first chance. Thanks for the follow up!

Reply:
Not a problem :)  It's just one of my pet issues: vetting the APIs we add wherever possible by trying to use them as best as possible in the millions of lines of production library code we maintain.

Reply:
I have opened https://github.com/dotnet/runtime/pull/79107.